shared/chat/conversation/messages/message-popup/text/index.tsx

```diff
@@ -105,6 +102,9 @@ const TextPopupMenu = (props: Props) => {
           },
         ]
       : []),
+    ...(props.onPinMessage
```

Comment:
pin is less useful than delete so i swapped them. on the SE sim delete is below the fold which is very annoying